### PR TITLE
Update top xml

### PR DESCRIPTION
**Pull request by lumpfish**
Changes for https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/132

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Tested here: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder-systemtest/165/

File: stf.samples/build.xml [hunk 1]
```diff
@@ -87,11 +92,11 @@ limitations under the License.
...
+		<mkdir dir="${stf_samples_bin_dir}"/>
 	</target>
 
 	<target name="clean">
-		<delete dir="${bin_dir}"/>
+		<delete dir="${stf_samples_bin_dir}"/>
 	</target>
 
-</project>
+</project>
```
Comment (Mesbah-Alam) [inline stf.samples/build.xml]: We should add a blank line at the end of this file.

File: stf.load/build.xml [hunk 2]
```diff
@@ -85,11 +90,11 @@ limitations under the License.
...
+		<mkdir dir="${stf_load_bin_dir}"/>
 	</target>
 
 	<target name="clean">
-		<delete dir="${bin_dir}"/>
+		<delete dir="${stf_load_bin_dir}"/>
 	</target>
 
-</project>
+</project>
```
Comment (Mesbah-Alam) [inline stf.load/build.xml]: We should add a blank line at the end of this file.

File: stf.core/build.xml [hunk 3]
```diff
@@ -98,11 +103,11 @@ limitations under the License.
...
 
 	<target name="create-bin-dir">
-		<mkdir dir="${bin_dir}"/>
+		<mkdir dir="${stf_core_bin_dir}"/>
 	</target>
 
 	<target name="clean">
-		<delete dir="${bin_dir}"/>
+		<delete dir="${stf_core_bin_dir}"/>
 	</target>
```
Comment (Mesbah-Alam) [inline stf.core/build.xml:112]: Blank line missing. We should add one at the end of this file.

File: stf.samples/build.xml (hunk 1 above)
Comment (lumpfish) [reply stf.samples/build.xml]: Fixed

File: stf.load/build.xml (hunk 2 above)
Comment (lumpfish) [reply stf.load/build.xml]: Fixed

Comment (lumpfish) [reply stf.load/build.xml]: Fixed

File: stf.core/build.xml (hunk 3 above)
Comment (lumpfish) [reply stf.core/build.xml:112]: Fixed

**Review by Mesbah-Alam — APPROVED** [review]
LGTM.

**Comment by Mesbah-Alam** [comment]
Grinder on Windows testing these PRs: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder-systemtest/169/console

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**